Lib/copy.py

```diff
@@ -177,6 +179,12 @@ def deepcopy(x, memo=None, _nil=[]):
         _keep_alive(x, memo) # Make sure x lives at least as long as d
     return y
 
+try:
+    from _copy import deepcopy
+except ImportError as ex:
```

Comment:
We never use `ex` so no point binding the exception to a local
```suggestion
except ImportError:
```